modules/shared/src/main/scala/org/tessellation/json/JsonBrotliBinarySerializer.scala

```diff
@@ -19,12 +19,21 @@ trait JsonBrotliBinarySerializer[F[_]] {
 object JsonBrotliBinarySerializer {
   private val compressionLevel = 2
 
-  def make[F[_]: Sync](): F[JsonBrotliBinarySerializer[F]] =
+  def apply[F[_]: JsonBrotliBinarySerializer]: JsonBrotliBinarySerializer[F] = implicitly
+
+  def forSync[F[_]: Sync]: F[JsonBrotliBinarySerializer[F]] = {
+    def printer = Printer(dropNullValues = false, indent = "")
+
+    forSync[F](printer)
+  }
+
+  def forSync[F[_]: Sync](printer: Printer): F[JsonBrotliBinarySerializer[F]] =
     Sync[F].delay(Brotli4jLoader.ensureAvailability()).map { _ =>
       new JsonBrotliBinarySerializer[F] {
+
         def serialize[A: Encoder](content: A): F[Array[Byte]] = {
           val params = new Parameters().setQuality(compressionLevel)
-          Sync[F].delay(BrotliEncoder.compress(content.asJson.noSpaces.getBytes("UTF-8"), params))
+          Sync[F].delay(BrotliEncoder.compress(content.asJson.printWith(printer).getBytes("UTF-8"), params))
```

Comment:
This may be a breaking change.

Reply:
it is not. it's using the same configuration, I just used an explicit way to render the json.
I'm using the following printer:
```scala
    def printer = Printer(dropNullValues = false, indent = "")
```

and `noSpaces` is configured as:
```scala
  final def noSpaces: String = Printer.noSpaces.print(this)

  /* --- */

  /**
   * A pretty-printer configuration that inserts no spaces.
   */
  final val noSpaces: Printer = Printer(
    dropNullValues = false,
    indent = ""
  )

```

Reply:
Ok great!